src/data/reducers/assets.js

```diff
@@ -40,20 +41,17 @@ const status = (state = {}, action) => {
   }
 };
 
-const types = (state = {}, action) => {
+const parameters = (state = initialState, action) => {
```

Comment:
I have absorbed the old types reducer into a new parameters reducer, which handles changes to the request parameters.

Reply:
I'm glad this now follows the same structure of the other reducers, it makes it more clear.